runtime/doc/autocmd.txt

```diff
@@ -261,6 +261,8 @@ Name			triggered by ~
 |SwapExists|		detected an existing swap file
 |TermOpen|		when a terminal job starts
 |TermClose|		when a terminal job ends
+|ChannelOpened|		after a channel opened
```

Comment:
I guess we should try to follow existing event name conventions: `ChanOpen` or `ChanOpenPost` . 

I dont think past tense is common.